File: JY-CH/baekjoon/b1330.py
# 두 수 비교하기

# a, b를 입력값으로 받을건데 정수형으로 받고 띄워서 받을거에요
a, b = map(int, input().split())
if a > b:
    print('>')
elif a < b:
    print('<')
else:
    print('==') # 남은 조건은 하나밖에 없어 입력하지 않아도 된다

File: JY-CH/Java/src/baekJoon/B1330.java
import java.util.Scanner;

public class B1330 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String input = scanner.nextLine();

        String[] numbers = input.split(" ");

        int a = Integer.parseInt(numbers[0]);
        int b = Integer.parseInt(numbers[1]);

        if (a > b) {
            System.out.print(">");
        }
        else if (a == b) {
            System.out.print("==");
        }
        else {
            System.out.print("<");
        }

    }
}
